r/run_model.R

```diff
@@ -0,0 +1,25 @@
+library(optparse)
+library(readr)
+library(tidyr)
+
+# Create a CLI ------------------------------------
+# Two args: `input_dir`, `output_dir`
+option_list <- list(
+  make_option("--input_dir", type="character", default="/input", metavar="character"),
+  make_option("--output_dir", type="character", default="/output", metavar="character")
+)
```

Comment:
I would make the arg names consistent with `README`. 
>python run_model.py --input-dir sample_data/ --output-dir 

^ I also check the python script, which seems that it has not supported input and output args yet?

Reply:
> not supported input and output args yet?

The Python script is using [`typer`](https://typer.tiangolo.com/tutorial/first-steps/#two-cli-arguments) to build the CLI!  I'm slowly moving towards this over `argparse`, as it offers a better development experience imo :-)

On that note, the Python script does use `input_dir` and `output_dir` as well, but it looks like `typer` converts the underscores in the arg names to dashes, which I didn't realize (found the [source code here](https://github.com/pallets/click/blob/dc918b48fb9006be683a684b42cc7496ad649b83/src/click/core.py#L2633)). If I were to do the same in the R script, I would have to replace `opt$input_dir` with `opt["input-dir"]`, right?

Reply:
> If I were to do the same in the R script, I would have to replace opt$input_dir with opt["input-dir"], right?

To access the value of item named "input-dir" (from a list), you can do 
```r
opt[["input-dir"]] 
# or 
opt$`input-dir`
````

Reply:
> The Python script is using [typer](https://typer.tiangolo.com/tutorial/first-steps/#two-cli-arguments) to build the CLI! I'm slowly moving towards this over argparse, as it offers a better development experience imo :-)

Ah, I realized I need to install `typer` locally first to be able to use the cli. 
Thanks for introducing `typer` - learn a new thing ✅

Reply:
Thanks!! Which approach is more commonly preferred? `opt[[...]]` or ``opt$`...` ``?

Reply:
Well, for most R codes I have seen, `opt[[...]]` is used more often than `` opt$`...` `` and I personally would prefer `opt[[...]]` as well.